posts/2021/08/low-code-workshop.md

```diff
@@ -0,0 +1,120 @@
+
+At Quansight Labs, we are motivated to provide opportunities for new contributor's to experience open source community work regardless of their software literacy. 
+Community workshops are a common format for onboarding, but sometimes the outcome can be less than satisfactory for participants and organizers. 
+In these workshops, there are implicit challenges that need to be overcome to contribute to projects revision history like Git or setting up development environments. 
+
+Our goal with the following low-code workshop is to offer a way for folks to join a projects contributors list without the technical overhead. 
+To achieve this we'll discuss a format that relies solely on the GitHub web interface.
+
+## Case study - collaboratively improving the accessibility of images in documentation
+
+Our successful experiments are based on having new and seasoned contributors work together to add alternative text representations to images in project documentation. 
+Each event lasts an hour with the following format:
+
+1. pre-meeting preparation with a project contributor and meeting facilitator
+2. a crash course in the specific topic - eg alt text for images
+3. an introduction to a collaborative pull request created by a project contributor
+4. group working session to suggest changes to specific files assigned in the pull request
+5. a short review of the submissions by the core contributor
+6. final review by the core contributor to submit a final pull request
+
+Accessibility is a focus for many members of Quansight Labs team so we've chosen this prompt to begin testing these workshops. 
+We'd love to hear about other prompts that may be good for low-code contributions.
+
+### Pre-meeting preparation
+
+Each event relies on a single pull request managed by a project contributor. 
+We recommend using a fork to reduce noise to other contributors because we gonna be loud for an hour. 
+
+Before the event the contributor makes a pull request identifying files that need changes related to the prompt, some projects may have rendered documentation that would also help guide the contributions. 
+The attendees will use these files to recommend changes to the code landing their aliases in the GitHub history. 
+The facilitator will add an agenda for the event with learning materials that will prepare folks for the event.
+
+### The crash course
+
+The crash course is a short lightning talk style introduction to support the prompt folks will address during the hour. 
+This short talk is important to prepare folks for the topic, to support contributors we want to remove the technical roadblocks that distract them from the community.
+
+### Introduction to the pull request 
+
+To kick off the work, the core developer walks us through the pull request they've prepared and introduce to the different files. 
```

Comment:
I'm not certain if you meant something else here, so my suggestion is a best guess at your thoughts.

```suggestion
To kick off the work, the core developer walks us through the pull request they've prepared and introduces different files. 
```